--- paste_form.php.orig	2019-11-26 09:57:23.151644359 +0100
+++ paste_form.php	2019-11-26 09:58:03.911858730 +0100
@@ -89,14 +89,11 @@
                                         "5" => lang('exp_5min'),
                                         "60" => lang('exp_1h'),
                                         "1440" => lang('exp_1d'),
-                                        "10080" => lang('exp_1w'),
-                                        "40320" => lang('exp_1m'),
-                                        "483840" => lang('exp_1y'),
 									);
                             if(! config_item('disable_keep_forever')) {
                                 $options['0'] = lang('exp_forever');
                             }
-						echo form_dropdown('expire', $options, $expire_set, $expire_extra); ?>
+						echo form_dropdown('expire', $options, 1440, $expire_extra); ?>
 					</div>
 				</div>
 			</div>
